### ISSUE TITLE
remove usage of couchr

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hoodie-plugin-users#35]
use hoodieAdmin request instead, for getConfig / setConfig. Problem is that we use the bearer token instead of cookies, so request will fail with a 401

Here's how the appconfig plugin is doing it.

```js
    function getConfig(callback) {
      hoodieAdmin.request('GET', '/app/config')
        .fail(function(error) { callback(error); })
        .done(function(response) { callback(null, response); })
    }

    function setConfig(doc, callback) {
      hoodieAdmin.request('PUT', '/app/config', {
        data: JSON.stringify(doc)
      })
        .fail(function(error) { callback(error); })
        .done(function(response) { callback(null, response); })
    }
```

**Comment by gr2m** [comment]
this is fixed via 2a8632b, but it's causing new issues ...

**Comment by espy** [comment]
Fixed in the new dashboard.
